### PR TITLE
Website to k8s.

### PR DESCRIPTION
**Pull request by svenvanhal**
Move website config to separate file, leave hendrik conf for flitcie / lists.

@praseodym is this the desired set-up?

### CODE REVIEW

File: config/website.conf [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    location /payments {
+        proxy_pass https://payments.k8s.chnet/payments;
+    }
+
+    location /choice2 {
+        proxy_pass https://choice.w3cie.k8s.chnet/choice2;
+    }
+
+    location / {
+        add_header Strict-Transport-Security "max-age=15768000; includeSubDomains; preload" always;
```
Comment (praseodym) [inline config/website.conf]: HSTS is already added by the Kubernetes ingress controller

File: config/website.conf [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    location /events {
+        proxy_pass https://events.k8s.chnet/events;
+    }
+
+    location /payments {
+        proxy_pass https://payments.k8s.chnet/payments;
+    }
+
+    location /choice2 {
+        proxy_pass https://choice.w3cie.k8s.chnet/choice2;
```
Comment (praseodym) [inline config/website.conf:50]: Is CHoice actually production-ready?

File: config/website.conf [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    }
+
+    location / {
+        add_header Strict-Transport-Security "max-age=15768000; includeSubDomains; preload" always;
+
+        proxy_ssl_server_name off;
+        proxy_set_header Host $host;
```
Comment (praseodym) [inline config/website.conf]: Does your application need this?

**Review by praseodym — CHANGES_REQUESTED** [review]

File: config/website.conf (hunk 2 above)
Comment (svenpopping) [reply config/website.conf:50]: Yes

File: config/website.conf (hunk 1 above)
Comment (svenpopping) [reply config/website.conf]: So it is not needed here?

Comment (praseodym) [reply config/website.conf]: Nope

Comment (svenvanhal) [reply config/website.conf]: ✅

File: config/website.conf (hunk 3 above)
Comment (svenvanhal) [reply config/website.conf]: Don't think so

**Review by praseodym — APPROVED** [review]

File: config/hendrik.conf [hunk 4]
```diff
@@ -1,63 +1,6 @@
 server {
     listen 443 ssl http2 default_server;
```
Comment (praseodym) [inline config/hendrik.conf:2]: Oops missed that

**Comment by praseodym** [comment]
We’re missing some more: any subdirectories at ch.tudelft.nl (like `/mail`) were implicitly handled by hendrik.chnet but are now all forwarded to the website. These specific subdirectories should be re-added.

**Pull request merged**